packs/equipment-effects/effect-caltrops.json

```diff
@@ -0,0 +1,46 @@
+{
+    "_id": "5DaEI4I7cVdOD507",
+    "img": "icons/weapons/thrown/bomb-spiked-purple.webp",
+    "name": "Effect: Caltrops",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Caltrops]</p>\n<p>While taking persistent bleed damage from caltrops, you take a –5-foot penalty to your Speed.</p>"
```

Comment:
You can grant the bleed with this and have it auto remove when the bleed is removed

```json
{
  "key": "GrantItem",
  "uuid": "Compendium.pf2e.conditionitems.Item.lDVqvLKA6eF3Df60",
  "alterations": [
    {
      "mode": "override",
      "property": "persistent-damage",
      "value": {
        "damageType": "bleed",
        "formula": "1"
      }
    }
  ],
  "onDeleteActions": {
    "grantee": "cascade"
  }
}
```